sirepo/server.py

```diff
@@ -254,7 +254,9 @@ def api_getApplicationData(filename=None):
     """
     req = http_request.parse_post(template=True, filename=filename or None)
     with simulation_db.tmp_dir() as d:
+        assert 'method' in req.req_data
```

Comment:
We should wrap get_application_data in something that does the checking of the inputs and outputs.